framework/src/utils/JsonSyntaxTree.C

```diff
@@ -18,6 +18,7 @@
 
 #include <algorithm>
 #include <cctype>
+#include <regex>
```

Comment:
I don't think we can merge this. The `<regex>` header isn't well supported with our minimum required compilers. Please use the pcre stuff in contrib for now. Someday...